src/quo2/components/switchers/base_card/view.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.components.switchers.base-card.view
+  (:require [quo.react-native :as rn]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.switchers.base-card.style :as style]))
+
+(defn base-card
+  [{:keys [banner on-press on-close customization-color]} & children]
+  (let [card-ref (atom nil)
+        set-ref  #(reset! card-ref %)]
+    [rn/touchable-opacity
```

Comment:
@codemaster115, this is not yet a valid form-2 component. The code I shared in this thread should be correct (although I didn't test) https://github.com/status-im/status-mobile/pull/16802#discussion_r1281283245

I suggest the following documentation if this whole form-2 stuff is not clear to you https://github.com/reagent-project/reagent/blob/a14faba55e373000f8f93edfcfce0d1222f7e71a/doc/CreatingReagentComponents.md#form-2--a-function-returning-a-function

Reply:
I'm sorry for the duplicate issue again. I will check the suggested documentation to be clear about form-2 component.
Thanks for your patience. @ilmotta

Reply:
No worries at all @codemaster115 :)

Reply:
also, don't we wish to start moving toward pressable `api` @ilmotta ?
https://reactnative.dev/docs/pressable